tutorials.md

```diff
@@ -1,19 +1,36 @@
 # Tutorials
+We've created a host of code-focused tutorials that will help you get
+started with *TimescaleDB*. 
+
+For most of these tutorials, you will also need a working instance of *TimescaleDB*. The fastest way
+to get started with *TimescaleDB* is [setting up *Timescale Cloud*][timescale-cloud-install].
+*Timescale Cloud* is a hosted version of *TimescaleDB*. If you prefer,
+you can download the [TimescaleDB Community Edition][timescale-community-install]
+and run it on your desktop or your own cloud infrastructure.
+
+### Common scenarios for using TimescaleDB
 
 - **[Start Here - Hello Timescale][Hello Timescale]**: If you are new to TimescaleDB
-or even SQL, check
-out our tutorial with NYC taxicab data to get an idea of the capabilities our
-database has to offer.
+or even SQL, check out our tutorial with NYC taxicab data to get an idea of the 
+capabilities our database has to offer.
 - **[Geospatial data][postGIS]**: If you want to delve into the further potential of
-TimescaleDB, we have guidance for more advanced topics
-like PostGIS integration.
+TimescaleDB, we have guidance for more advanced topics like PostGIS integration.
```

Comment:
Just a note: With the redesign of the Hello Timescale tutorial, the place where this links to is not really a good place to jump the reader to, as it's not really a standalone section anymore (and it was always a little weird to jump the reader into the middle of another tutorial).  If possible, it would be a good idea to make a separate PostGIS tutorial, even something short; or make the PostGIS section of Hello Timescale into it's own clearly distinct section.

Reply:
I might suggest in the interim to remove this link altogether.